SEP-0010.md

```diff
@@ -0,0 +1,33 @@
+# SEP-0009 -- Release Pattern for SunPy Core
+
+| SEP           | 10                              |
+|---------------|--------------------------------|
+| title         | Mission and Vision Statements |
+| author(s)     | |
+| contact email |               |
+| date-creation | 2020-09-08                     |
+| type          | Informational                        |
+| discussion    |  |
+| status        |                       |
+
+# Introduction
+
+The purpose of this SEP is to formally define a mission and vision statement for the SunPy Project.
+The purpose of these two statements is to form a foundation upon which the actions of the Project can stand.
+
+The vision statement is intended as a guide to help the project make decisions that align with its philosphy.
+A vision statement is meant to be inspirational and idealistic.
+It is meant to describe a future that the Project is trying to make a reality.
+
+The mission statement is a formal summary of the goals of the Project.
+It describes what the Project is trying to achieve.
+It should be consistent with the vision statement.
+
+# Vision Statement
+A diverse and inclusive solar physics and heliophysics community that supports scientific discovery and reproducibility through the development of accessible, open-source software.
```

Comment:
Can we combine "solar physics and heliophysics" to have one less physics?

I also feel that starting with A feels a bit passive.

Reply:
People wanted solar physics mentioned in particular, maybe we just remove heliophysics? But i remember folks saying that we wanted to think more broadly than just solar.

Reply:
I am pro it being more broad, I wondering if we could combine both into one fewer word.